src/pages/Hacker/Pass.tsx

```diff
@@ -57,69 +49,82 @@ const HackPassWrapper = styled.div`
     }
   }
 
-  button[type="submit"] {
+  button[type='submit'] {
```

Comment:
lol

Reply:
oh I think that's Prettier doing it's work